spritz/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        jcenter()
+
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.3'
+
+        // NOTE: Do not place your application dependencies here; they belong
```

Comment:
Could remove these comments

Reply:
Done in 2f5e61a472fd70180ccdad9a6c9ea95810b4dd75.